src/cli.ts

```diff
@@ -328,18 +343,20 @@ async function main() {
       const manifest = JSON.parse(manifestText);
       currentVersion = manifest.version;
     } catch (error) {
-      if (error.code === 'ENOENT') {
-        return exitWithError(
-          `Package manifest not found at path: '${manifestPath}'\nRun this script from the project root directory, or set the project directory using the '--root' flag.`,
-        );
-      } else if (error.code === 'EACCES') {
-        return exitWithError(
-          `Access to package manifest is forbidden by file access permissions: '${manifestPath}'`,
-        );
-      } else if (error.name === 'SyntaxError') {
-        return exitWithError(
-          `Package manifest cannot be parsed as JSON: '${manifestPath}'`,
-        );
+      if (hasErrorCode(error)) {
+        if (error.code === 'ENOENT') {
+          return exitWithError(
+            `Package manifest not found at path: '${manifestPath}'\nRun this script from the project root directory, or set the project directory using the '--root' flag.`,
+          );
+        } else if (error.code === 'EACCES') {
+          return exitWithError(
+            `Access to package manifest is forbidden by file access permissions: '${manifestPath}'`,
+          );
+        } else if (error.name === 'SyntaxError') {
+          return exitWithError(
+            `Package manifest cannot be parsed as JSON: '${manifestPath}'`,
+          );
+        }
```

Comment:
Shouldn't this be moved to outside the `hasErrorCode` block?

Reply:
Ah, good catch, I will fix this

Reply:
Done in d16296b63f1da4bb098f834230f52821dc2ad5a0